bundles/AdminBundle/Controller/Admin/ElementController.php

```diff
@@ -774,4 +745,77 @@ public function analyzePermissionsAction(Request $request)
             ]
         );
     }
+
+    /**
+     * @param DataObject\Concrete $source
+     * @param                     $context
+     * @param                     $result
+     * @param                     $targets
+     * @return array
+     * @throws \Exception
+     */
+    protected function convertResultWithPathFormatter(DataObject\Concrete $source, $context, $result, $targets): array
+    {
+        $ownerType = $context['containerType'];
+        $fieldname = $context['fieldname'];
+        $fd = null;
+
+        if ($ownerType == 'object') {
+            $fd = $source->getClass()->getFieldDefinition($fieldname);
+        } elseif ($ownerType == 'localizedfield') {
+            $fd = $source->getClass()->getFieldDefinition('localizedfields')->getFieldDefinition($fieldname);
+        } elseif ($ownerType == 'objectbrick') {
+            $fdBrick = DataObject\Objectbrick\Definition::getByKey($context['containerKey']);
+            $fd = $fdBrick->getFieldDefinition($fieldname);
+        } elseif ($ownerType == 'fieldcollection') {
+            $containerKey = $context['containerKey'];
+            $fdCollection = DataObject\Fieldcollection\Definition::getByKey($containerKey);
+            if ($context['subContainerType'] == 'localizedfield') {
+                $fdLocalizedFields = $fdCollection->getFieldDefinition('localizedfields');
+                $fd = $fdLocalizedFields->getFieldDefinition($fieldname);
+            } else {
+                $fd = $fdCollection->getFieldDefinition($fieldname);
+            }
+        }
+
+
+        if ($fd instanceof DataObject\ClassDefinition\PathFormatterAwareInterface) {
```

Comment:
@dpfaffenbauer can you please explain..how this suppose to work? afaik there are no field definitions implementing this interface.

Reply:
guess it got lost during the rebase, I re-added it

Reply:
please also fix the exception. thanks! `Pimcore\Model\DataObject\ClassDefinition\Data\Relations\AbstractRelations::getPathFormatterClass() must be compatible with Pimcore\Model\DataObject\ClassDefinition\PathFormatterAwareInterface::getPathFormatterClass(): ?string`

Reply:
done